demo/server.js

```diff
@@ -151,3 +152,24 @@ initSqlJs().then(SQL => {
 
   app.listen(port, () => console.log(`Listening on port ${port}`));
 });
+
+ // Get the accuracy history for a model
+ function getAccuracyHistory()
+ {
+
+ }
+ 
+ // Add a new accuracy record for a model
+  function addAccuracyRecord(accuracy)
+  {
+   db.run('INSERT INTO accuracy VALUES (?, ?, ?, ?);', [
+     accuracy.transaction_hash,
+     accuracy.block_number,
```

Comment:
These keys should be camelCase because the come JavaScript or JSON in an API.

Reply:
Done !